src/index.html

```diff
@@ -8,6 +8,13 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars_item">
+      <li class="stars stars__0"></li>
+      <li class="stars stars__1"></li>
+      <li class="stars stars__2"></li>
+      <li class="stars stars__3"></li>
+      <li class="stars stars__4"></li>
+      <li class="stars stars__5"></li>
+    </ul>
```

Comment:
```suggestion
<div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
      <div class="stars stars--1">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
      
      ....
```
Тут має бути 6 секцій з зірками з різними модифікаторами: stars stars--0, stars stars--1, stars stars--2 і т.д.
Від назви модифікатора має залежати, скільки зірок будуть відобрадатись активними